lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -99,7 +99,27 @@ class ReportUIFeatures {
     window.addEventListener('resize', this._updateStickyHeaderOnScroll);
     this._setupMetricDescriptionToggleElements();
     const topbarLogo = this._dom.find('.lh-topbar__logo', this._document);
-    topbarLogo.addEventListener('click', this._toggleDarkTheme);
+    topbarLogo.addEventListener('click', () => this._toggleDarkTheme());
```

Comment:
ts error here ... How do I allow for undefined param?

```
An argument for 'force' was not provided.
```

Reply:
`@param {boolean} [force]`

Reply:
I think @brendankenny prefers `{T=}` which might have the same effect though?